Common/src/CConfig.cpp

```diff
@@ -4332,6 +4332,12 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     }
   }
 
+  if(Kind_TimeIntScheme_Turb==RUNGE_KUTTA_EXPLICIT ||
+     Kind_TimeIntScheme_Turb==CLASSICAL_RK4_EXPLICIT ||
+     Kind_TimeIntScheme_Turb==ADER_DG){
+    SU2_MPI::Error("Only TIME_DISCRE_FLOW = EULER_IMPLICIT, EULER_EXPLICIT have been implemented.", CURRENT_FUNCTION);
```

Comment:
Not sure about the ADER_DG part
```suggestion
    SU2_MPI::Error("Only TIME_DISCRE_TURB = EULER_IMPLICIT, EULER_EXPLICIT have been implemented.", CURRENT_FUNCTION);
```